content/en/language_clients/python/verifying.md

```diff
@@ -0,0 +1,160 @@
+                        bundle (default: False)
+  --cert-identity IDENTITY
+                        The identity to check for in the certificate's Subject
+                        Alternative Name (default: None)
+  --cert-oidc-issuer URL
+                        The OIDC issuer URL to check for in the certificate's
+                        OIDC issuer extension (default: None)
+```
+<!-- @end-sigstore-verify-identity-help@ -->
+
+### Signatures from GitHub Actions
+
+If your signatures are coming from GitHub Actions (e.g., a workflow
+that uses its [ambient credentials](../signing#signing-with-ambient-credentials)),
+then you can use the `sigstore verify github` subcommand to verify
+claims more precisely than `sigstore verify identity` allows:
+
+<!-- @begin-sigstore-verify-github-help@ -->
+```console
+usage: sigstore verify github [-h] [-v] [--certificate FILE]
+                              [--signature FILE] [--bundle FILE] [--offline]
+                              [--cert-identity IDENTITY] [--trigger EVENT]
+                              [--sha SHA] [--name NAME] [--repository REPO]
+                              [--ref REF]
+                              FILE [FILE ...]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -v, --verbose         run with additional debug logging; supply multiple
+                        times to increase verbosity (default: 0)
+
+Verification inputs:
+  --certificate FILE, --cert FILE
+                        The PEM-encoded certificate to verify against; not
+                        used with multiple inputs (default: None)
+  --signature FILE      The signature to verify against; not used with
+                        multiple inputs (default: None)
+  --bundle FILE         The Sigstore bundle to verify with; not used with
+                        multiple inputs (default: None)
+  FILE                  The file to verify
+
+Verification options:
+  --offline             Perform offline verification; requires a Sigstore
+                        bundle (default: False)
+  --cert-identity IDENTITY
+                        The identity to check for in the certificate's Subject
+                        Alternative Name (default: None)
+  --trigger EVENT       The GitHub Actions event name that triggered the
+                        workflow (default: None)
+  --sha SHA             The `git` commit SHA that the workflow run was invoked
+                        with (default: None)
+  --name NAME           The name of the workflow that was triggered (default:
+                        None)
+  --repository REPO     The repository slug that the workflow was triggered
+                        under (default: None)
+  --ref REF             The `git` ref that the workflow was invoked with
+                        (default: None)
+```
+<!-- @end-sigstore-verify-github-help@ -->
+
+## Advanced verification use cases
```

Comment:
It looks like this section might be a bit out of date compared to the current README, I was looking for these sections and couldn't find them:

https://github.com/sigstore/sigstore-python/tree/main?tab=readme-ov-file#verifying-against-a-bundle (previously called "Verifying against a signature and certificate"?)
https://github.com/sigstore/sigstore-python/tree/main?tab=readme-ov-file#offline-verification
https://github.com/sigstore/sigstore-python/tree/main?tab=readme-ov-file#verifying-a-digest-instead-of-a-file

Reply:
Yeah, we're removing/deprioritizing these since detached material support is on the chopping block -- we're likely going to require bundles only with the next major release. https://github.com/sigstore/sigstore-python/issues/1183 has some more context there 🙂